### PR TITLE
A new site block has been added in accordance with the design describ…

### PR DESCRIPTION
**Pull request by petrovsky747**
…ed in Figma.

To build the html markup of the block used unordered list (in order to simplify the viewing of content for people with disabilities).
A style reset was applied while developing the stylesheet (taken from here https://meyerweb.com/eric/tools/css/reset/).
Classes were defined for each type of objects (the BEM methodology was applied for naming) to customize the display of content on the page.

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+}
+.advantages__title {
+    text-align: center;
+    /*font-family: 'Roboto';*/
+    font-style: normal;
+    font-weight: 500;
+    font-size: 40px;
+    margin: 120px auto 20px;
+    /*line-height: 47px;*/
+    /* identical to box height */
```
Comment (5tronciy) [inline styles/styles.css]: Don't commit the useless comments.

File: styles/styles.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+    content: '';
+    content: none;
+}
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
```
Comment (5tronciy) [inline styles/styles.css:4]: You may move the code above to other file, for example reset.css

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Looks good, but there is no images.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Good in general.
And why you didn't use fonts from the design? Did you ask about it at the weekly meeting?

**Comment by petrovsky747** [comment]
> Good in general. And why you didn't use fonts from the design? Did you ask about it at the weekly meeting?

During the 4th lesson, the mentor pointed out that we will go through the fonts in the next lesson and they can not be connected to lesson-4 :)

**Comment by 5tronciy** [comment]
> > Good in general. And why you didn't use fonts from the design? Did you ask about it at the weekly meeting?
> 
> During the 4th lesson, the mentor pointed out that we will go through the fonts in the next lesson and they can not be connected to lesson-4 :)

Ok, let's do it next time)

**Pull request merged**